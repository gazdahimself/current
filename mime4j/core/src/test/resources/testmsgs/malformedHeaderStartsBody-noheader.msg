This is a simple message not having headers.
The whole text should be recognized as body.
