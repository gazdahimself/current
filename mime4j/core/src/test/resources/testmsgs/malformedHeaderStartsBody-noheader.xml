<message>
<header>
</header>
<body>
This is a simple message not having headers.
</body>
</message>
