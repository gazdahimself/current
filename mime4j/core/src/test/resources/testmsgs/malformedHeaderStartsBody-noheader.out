
This is a simple message not having headers.
